src/entt/signal/dispatcher.hpp

```diff
@@ -248,6 +253,31 @@ class dispatcher {
         }
     }
 
+    /**
+     * @brief Returns the count of pending events for the specified type.
+     *
+     * This method returns the count of pending events for the specified
+     * type.
+     */
+    template<typename Event>
+    size_t size() const {
+        return assure<Event>().size();
+    }
+
+    /**
+     * @brief Returns the total count of pending events.
+     *
```

Comment:
```suggestion
     * @return The total count of pending events.
```
Please, also remove the comment below.